### PR TITLE
Dimagisphere tweaks

### PR DESCRIPTION
**Pull request by czue**
@benrudolph, some of the changes I mentioned. can read :blowfish: / :blowfish: 

main thing it adds are a little hover popup and a "simulate" button, as well as a bit of code reorg.

cc @orangejenny

### CODE REVIEW

File: corehq/apps/hqadmin/static/hqadmin/js/dimagisphere_helper.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    self.generateRandomItem = function () {
+        // just return a random item from FAKE_DOMAINS
+        // http://stackoverflow.com/questions/2532218/pick-random-property-from-a-javascript-object
+        var domains = Object.keys(FAKE_DOMAINS);
+        var randomDomain = domains[domains.length * Math.random() << 0];
+        return {
+            domain: randomDomain,
+            country: FAKE_DOMAINS[randomDomain]
+        }
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/dimagisphere_helper.js]: Missing semicolon.

**Comment by czue** [comment]
(no test coverage)

**Comment by benrudolph** [comment]
looks good! will let you respond if you want, otherwise g2m

**Comment by czue** [comment]
:tada:

**Comment by czue** [comment]
:earth_africa: :earth_asia: :earth_americas:

**Comment by benrudolph** [comment]
:full_moon_with_face:

**Pull request merged**